var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -109,8 +113,6 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("coinhsl+blas", when="+sparse")
     depends_on("metis", when="+sparse")
 
-    depends_on("ginkgo@glu+cuda", when="+ginkgo")
```

Comment:
We need to keep Ginkgo dependency.

Reply:
I have re-added this. Thanks for catching